tidb-cloud/set-up-private-endpoint-connections-on-azure.md

```diff
@@ -0,0 +1,121 @@
+
+## Restrictions
+
+- Only the `Organization Owner` and `Project Owner` roles can create private endpoints.
+- The private endpoint and the TiDB cluster to be connected must be located in the same region.
+
+## Set up a private endpoint with Azure Private Link
+
+To connect to your TiDB Cloud Dedicated cluster via a private endpoint, complete the following steps:
+
+1. [Select a TiDB cluster](#step-1-select-a-tidb-cluster)
+2. [Create an Azure private endpoint](#step-2-create-an-azure-private-endpoint)
+3. [Accept the endpoint](#step-3-accept-the-endpoint)
+4. [Connect to your TiDB cluster](#step-4-connect-to-your-tidb-cluster)
+
+If you have multiple clusters, you need to repeat these steps for each cluster that you want to connect to using Azure Private Link.
+
+### Step 1. Select a TiDB cluster
+
+1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click the name of your target TiDB cluster to go to its overview page.
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+3. In the **Connection Type** drop-down list, select **Private Endpoint**, and then click **Create Private Endpoint Connection** to open the **Create Azure Private Endpoint Connection** dialog.
+
+> **Note:**
+>
+> If you have already created a private endpoint connection, the active endpoint will appear in the connection dialog. To create additional private endpoint connections, navigate to the **Networking** page in the left navigation pane.
+
+### Step 2. Create an Azure private endpoint
+
+1. In the **Create Azure Private Endpoint Connection** dialog, copy the TiDB Cloud resource ID of the private link service and leave the dialog open for later use.
+
+    > **Note:**
+    >
+    > For each TiDB Cloud Dedicated cluster, the corresponding endpoint service is automatically created 3 to 4 minutes after the cluster creation.
+
+2. Log in to the [Azure portal](https://portal.azure.com/), and then create a private endpoint for your cluster using the copied TiDB Cloud resource ID as follows:
+
+    1. In the Azure portal, search for **Private endpoints**, and then select **Private endpoints** in the result.
+    2. On the **Private endpoint** page, click **+ Create**.
+    3. In the **Basics** tab, fill in the project and instance information, and then click **Next: Resource**.
+    4. In the **Resource** tab, choose **Connect to an Azure resource by resource ID or alias** as the **connection method**, paste the TiDB Cloud resource ID to the **Resource ID or alias** field, and then click **Next: Virtual Network**.
+    5. Complete the rest of the configurations of this private endpoint, and then click **Create**. For more information, see [Create a private endpoint](https://learn.microsoft.com/en-us/azure/private-link/create-private-endpoint-portal?tabs=dynamic-ip#create-a-private-endpoint) in Azure documentation.
```

Comment:
```suggestion
    4. In the **Resource** tab, choose **Connect to an Azure resource by resource ID or alias** as the **connection method**, and paste the TiDB Cloud resource ID to the **Resource ID or alias** field.
    5. Continue clicking **Next** to go through the remaining configuration tabs and complete the required settings. Then, click **Create** to create and deploy the private endpoint. It might take a few seconds for Azure to complete the deployment. For more information, see [Create a private endpoint](https://learn.microsoft.com/en-us/azure/private-link/create-private-endpoint-portal?tabs=dynamic-ip#create-a-private-endpoint) in Azure documentation.
```